views/layouts/_footer.html.haml

```diff
@@ -0,0 +1,116 @@
+%footer.footer.pt-3.pb-3
+  - portal_base = Mp::Application.config.portal_base_url
+  .container
+    .row.h-100
+      .col-md-3.my-auto
+        %a.egi-logo-mono.d-block{ href: portal_base }
+          = image_pack_tag "egi-footer-logo.png"
+      .col-md-9.my-auto.copyright
+        %span.copy-text
+          Copyright 2018 - All right reserved
+        %a.ml-4.text-uppercase{ href: "https://eosc-portal.eu/privacy-policy-summary" }
+          privacy policy
+    %hr.mb-4
+    .row
+      .col-md
+        %ul
+          %li
+            .title About
+          %li.mb-1
+            %a{ href: "#{portal_base}/about/eosc" } EGI
+          %li.mb-1
+            %a{ href: "#{portal_base}/about/eosc-doers" } EGI Doers
```

Comment:
I suppose these links should be fixed by e.g. environmental variable. For the first public show i think it's ok but we need to refactor that